Xamarin.Forms.Platform.UAP/CollectionView/ItemsViewRenderer.cs

```diff
@@ -117,21 +117,18 @@ protected virtual void CleanUpCollectionViewSource()
 				{
 					observableItemTemplateCollection.CleanUp();
 				}
-			}
 
-			if (Element?.ItemsSource == null)
-			{
 				if (CollectionViewSource?.Source is INotifyCollectionChanged incc)
 				{
 					incc.CollectionChanged -= ItemsChanged;
 				}
 
-				if (CollectionViewSource != null)
-				{
-					CollectionViewSource.Source = null;
-				}
-
+				CollectionViewSource.Source = null;		
```

Comment:
Won't this throw an exception is CollectionViewSource is `null`?

Reply:
Yes, but there is a null check in the line 114.